packages/gene-tools/src/generators/nextjs-app-generator/index.spec.ts

```diff
@@ -239,30 +223,6 @@ describe('NextJS App generator', () => {
     expect(eslintJSON.settings.next.rootDir).toBe(appDir);
   });
 
-  it('.eslintrc for e2e should contain updated overrides', async () => {
```

Comment:
I believe this was valid use case

Reply:
New NX generator does not generate this file at all

Reply:
actually nevermind, it was the case in one of the NX versions, I will readd this